ResearchKit/Common/ORKImageCaptureStep.m

```diff
@@ -88,4 +88,12 @@ - (ORKPermissionMask)requestedPermissions {
     return mask | ORKPermissionCamera;
 }
 
+- (void)validateParameters{
+    [super validateParameters];
+    
+    if ( self.templateImage != nil && self.templateImageAccessibilityHint == nil) {
```

Comment:
I would argue against this entire function.   A template image is NOT required in order to use the ``ORKImageCaptureStep`` and the existing accessibility hint is always applied to the preview view, not only when it has a valid template image.